infra/modules/main/cloudwatch.tf

```diff
@@ -45,6 +45,18 @@ resource "aws_cloudwatch_dashboard" "overview" {
                         {
                             "label": "Second result articles (https://www.hs.fi/kotimaa/art-2000006463218.html & https://www.is.fi/kotimaa/art-2000006463005.html)",
                             "value": "2020-04-03T16:00:00+03:00"
+                        },
```

Comment:
Now what is this dashboard?

Reply:
You can enjoy it [here in our AWS console](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#dashboards:)!